examples/bin/druid-quickstart.py

```diff
@@ -0,0 +1,337 @@
+
+def build_supervise_script_arguments(service_list, service_memory_config, root_config_path, run_zk):
+    argument_list = []
+
+    argument_list.append("\":verify bin/verify-java\"")
+    argument_list.append("\":verify bin/verify-default-ports\"")
+    argument_list.append("\":notify bin/greet\"")
+    argument_list.append("\":kill-timeout 10\"")
+
+    if run_zk:
+        argument_list.append("\"!p10 zk bin/run-zk conf\"")
+
+    for service in service_list:
+        prefix = ''
+        if service == MIDDLE_MANAGER_SERVICE_NAME:
+            prefix = '!p90 '
+
+        jvm_args = service_memory_config.get(service)
+
+        if jvm_args is None:
+            argument_list.append(f'\"{prefix}{service} bin/run-druid {service} {root_config_path}\"')
+        else:
+            argument_list.append(f'\"{prefix}{service} bin/run-druid {service} {root_config_path} \'{jvm_args}\'\"')
+
+    print('Supervise script command:')
+    for item in argument_list:
+        print(item)
+
+    print('\n')
+
+    return ",".join(argument_list)
+
+def display_help():
+    text = """
+    Usage: start-druid [options]
+
+    where options include:
+       totalMemory=<value>
+            Integer value is supported with 'm' or 'g' suffix.
+            Memory for druid services, if totalMemory is not specified
+            80 percent of system memory is used.
+            Note, if service specific jvm config is present,
+            <totalMemory> shouldn't be specified.
+            <totalMemory> should be greater than equals 3g
+       rootConfigPath=<value>
+            Directory containing common and service specific
+            properties to be overridden, this directory must contain '_common'
+            directory with 'common.jvm.config' & 'common.runtime.properties'
+            files.
+            If <rootConfigPath> is not specified, config from
+            'conf/druid/single-server/quickstart' directory is used.
+            This path is relative to current working directory
+       services=<value>
+            Value is comma separated string.
+            List of services to be started, should be a subset of
+            [broker, router, middleManager, historical, coordinator-overlord].
+            If runtime or jvm properties are to be overridden, they should be
+            kept within <rootConfigPath>/<service>.
+            If <rootConfigPath> is not specified config files can also be
+            placed within `conf/druid/single-server/quickstart/<service>`
+            directory.
+            Note, if jvm.config file is present for one of the services,
+            it must be present for all services.
+            If <services> argument is not given, all services
+            alongwith zookeeper is started.
+       runZk
+            Specification to run zookeeper, zk config is picked up from conf/zk.
+       computeOnly
+            Validate the arguments and display memory distribution for services.
+
+    sample usage:
```

Comment:
List out the example usages first, then explain the parameters in detail. The examples should be self-explanatory, so I don't think we need a description for each. If we do include a description, it should not be more than one line.

Reply:
`argparse` lib is generating the help documentation now.